Python/bytecodes.c

```diff
@@ -167,12 +167,32 @@ dummy_func(
             ERROR_IF(res == NULL, error);
         }
 
+        inst(UNARY_POSITIVE_R, (--)) {
+            PyObject *value = REG(oparg1);
+            assert(value != NULL);
+            PyObject *res = PyNumber_Positive(value);
+            PyObject *tmp = REG(oparg2);
+            REG(oparg2) = res;
+            Py_XDECREF(tmp);
+            ERROR_IF(res == NULL, error);
```

Comment:
Why destroy the register before jumping to the error label?

Also, Isn't there a macro `Py_SETREF()` that does the little dance with the temp variable and the XDECREF?

Reply:
Thanks, I forgot what it was called. Everything here is now on the [new PR](https://github.com/python/cpython/pull/100276) (on today's main). So I can close this one and we only have one on the go.